cpp/include/cudf/types.h

```diff
@@ -258,8 +269,16 @@ typedef struct gdf_context_{
   int flag_distinct;            ///< for COUNT: DISTINCT = 1, else = 0
   int flag_sort_result;         ///< When method is GDF_HASH, 0 = result is not sorted, 1 = result is sorted
   int flag_sort_inplace;        ///< 0 = No sort in place allowed, 1 = else
+  bool flag_groupby_include_nulls; 
+                                /**< false = Nulls are ignored in group by keys (Pandas style), 
+                                true = Nulls are treated as values in group by keys where NULL == NULL (SQL style)*/ 
+  gdf_nulls_sort_behavior flag_null_sort_behavior; 
+                                /**< GDF_NULL_AS_LARGEST = Nulls are treated as largest,
```

Comment:
I don't think you need to document the meaning of each enum option here, since they are documented in the enum. Just say `Sort behavior for null elements` or something concise.

Reply:
Fixed